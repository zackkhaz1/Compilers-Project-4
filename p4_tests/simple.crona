glob:bool;
main:int(){
	a:int;
	glob = a;
}
