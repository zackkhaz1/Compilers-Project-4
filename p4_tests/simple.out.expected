glob(bool):bool;
main(->int):int(){
	a(int):int;
	glob(bool) = a(int);
}
